.github/workflows/test.yml

```diff
@@ -26,6 +26,9 @@ jobs:
     - uses: dtolnay/rust-toolchain@master
       with:
         toolchain: ${{ matrix.rust }}
+    - uses: Swatinem/rust-cache@v2
+      with:
+        key: ${{ matrix.profile }}
```

Comment:
Why is the explicit key necessary? My reading of the documentation is that it's already implicitly included, no?

Reply:
Can you show me where you read that? I don't think it can be implicitly included since each project may use a different key for this.

Reply:
It was my reading of [this paragraph](https://github.com/Swatinem/rust-cache/blob/b1db5f9d5f9ed685de3a2774c3d3b838d2dde462/README.md?plain=1#L26-L27):
>     # An additional cache key that is added alongside the automatic `job`-based
>     # cache key and can be used to further differentiate jobs.

But then it's not clear what the "`job`-based cache key" is. I took it as something derived from the job's name (which does include the profile)

Reply:
I don't think the job name is used; only the [job_id](https://github.com/Swatinem/rust-cache/blob/b1db5f9d5f9ed685de3a2774c3d3b838d2dde462/README.md?plain=1#L108C58-L108C58). I believe that means it's shared across the matrix.

Reply:
Gotcha. Makes sense then.